subprojects/docs/src/docs/userguide/authoring-builds/gradle_init.adoc

```diff
@@ -0,0 +1,103 @@
+// Copyright 2023 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[gradle_init]]
+= Gradle Init
+
+For users that are starting a new project, Gradle comes with a built-in task called `init` that initializes a Gradle project.
+
+This ensures that you have the minimum set of requirements for Gradle to work when developing a new app, plugin, or library.
+
+The `init` task uses the built-in `wrapper` task to create a Gradle wrapper script, `gradlew` and `gradle.bat`.
```

Comment:
I think its worth mentioning that the wrapper configuration will be set to the version of Gradle that was used when executing the init task